src/js/component.js

```diff
@@ -358,8 +359,6 @@ class Component {
       return;
     }
 
-    name = toTitleCase(name);
```

Comment:
so, basically, this is the big gain here? Because we register the component both as `component` and as `Component`, we can just assume it'll be available and we don't need to toTitleCase the name?

Reply:
exactly!